libs/core-styles/src/lib/_imports/settings/color.css

```diff
@@ -56,4 +56,8 @@
 
   /* Generic */
   --global-color-background--app: var(--global-color-primary--xx-light);
+
+  /* Link */
+  --global-color-link-on-light--normal: var(--global-color-accent--normal);
+  --global-color-link-on-dark--normal: var(--global-color-accent--light);
```

Comment:
ℹ️ Copied from CMS, now that [they are needed outside CMS](https://github.com/TACC/tup-ui/pull/47/files#diff-4cd7d0a0a99e5a24250d60b0a088ddac76bcbb38530bef59c83f61b0a3e0e27dR174-R177).